torch/functional.py

```diff
@@ -331,9 +331,33 @@ def meshgrid(*tensors: Union[Tensor, List[Tensor]]) -> Tuple[Tensor, ...]:
         return _meshgrid(*tensors)
 else:
     def meshgrid(*tensors):
-        r"""Take :math:`N` tensors, each of which can be either scalar or 1-dimensional
-        vector, and create :math:`N` N-dimensional grids, where the :math:`i` :sup:`th` grid is defined by
-        expanding the :math:`i` :sup:`th` input over dimensions defined by other inputs.
+        r"""Creates grids of coordinates specified by the input tensors.
+
+        Take :math:`N` tensors :math:`T_0, T_1, \ldots, T_N`, each of
+        which can be a scalar* or a vector. Produces :math:`N`
+        N-dimensional tensors, each of shape :math:`(|T_0|, |T_1|
+        \ldots |T_n|)`. Each tensor output :math:`G_i` is :math:`T_i`
+        in the :math:`i` :sup:`th` dimension, expanded over the other
```

Comment:
I'm not totally sure what "expanded over the other dimensions means."

Reply:
I acknowledge this is clear, but I don't know how to articulate this better. I quite literally mean the "expand" operator, where what is expanded is that particular input vector in the corresponding dimension in the grid. But that is language derived from the implementation itself.

Reply:
Maybe just a parenthetical with a code snippet, then?

Reply:
moot now that i've stolen your words. thanks btw :D